hack/vcluster/partials/main.go

```diff
@@ -105,8 +105,8 @@ func main() {
 
 	util.DefaultRequire = false
 	versionDir := os.Args[1]
-	jsonSchemaPath := filepath.Join(versionDir, "vcluster.schema.json")
-	defaultValues := filepath.Join(versionDir, "default_values.yaml")
+	jsonSchemaPath := filepath.Join(versionDir, "values.schema.json")
```

Comment:
Probably I'm missing some context, but in order it to work with the vcluster repo I had to rename those files 🤔. (cc @Piotr1215 @joaniekube)

Reply:
@hidalgopl can you please confirm? I'm not sure if the files have to be renamed.

Reply:
yeah, renaming these will break the CI that updates partials.

Could you describe @johannesfrey what are you trying to do and what did not work?

Reply:
Ah, didn't know that we have a CI step for this here in vcluster. So I guess I can delete this PR as the content here will be a auto generated PR when the next beta/rc is created?